test/common/network/cidr_range_test.cc

```diff
@@ -68,7 +68,6 @@ TEST(TruncateIpAddressAndLength, Various) {
       {{"ffff::ffff", 128}, {"ffff::ffff", 128}},
       {{"ffff::ffff", 999}, {"ffff::ffff", 128}},
   };
-  test_cases.size();
```

Comment:
Why was this there before? Was it testing to ensure that an exception wasn't thrown? Or just an accidental inclusion?

Reply:
There's a google test macro for that, so I'd presume it was laziness after development, perhaps?